src/components/map/LocationOverview.tsx

```diff
@@ -96,6 +99,7 @@ export const LocationOverview = ({
   const [selectedPointOfInterest, setSelectedPointOfInterest] = useState<string>();
   const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp });
   const [isLocationAlertShow, setIsLocationAlertShow] = useState(false);
+  const isPreviewWithNavigation = route.params?.isPreviewWithNavigation ?? true;
```

Comment:
i think it should be `isPreviewWithoutNavigation` here. the default is that there is a navigation. so if someone wants to have it differently, then a "please do not navigate" can be activated and set to `true`, which is `false` by default. i find it less logical and less readable that i have to explicitly set a "please navigate" to "no", as it would currently be the case. what do you think about flipping it?

Reply:
Yes, it makes sense. I'll do that.

Reply:
[65bad2c](https://github.com/smart-village-solutions/smart-village-app-app/pull/1277/commits/65bad2ca8bcd494ac4a41a69b4d6a6126657033a)